server_admin/topics/clients/client-oidc.adoc

```diff
@@ -123,10 +123,29 @@ See link:{adapterguide_link}[{adapterguide_name}] for more information.
 To fill in the `Web Origins` data, enter in a base URL and click the + sign to add.  Click the - sign next to URLs you want to remove.
 Remember that you still have to click the `Save` button!
 
+==== Advanced Settings
 
+[[_mtls-client-certificate-bound-tokens]]
+*OAuth 2.0 Mutual TLS Client Certificate Bound Access Token*
 
+Mutual TLS binds an access token and a refresh token with a client certificate exchanged during TLS handshake. This prevents an attacker who finds a way to steal these tokens from exercising the tokens. The type of token is called a holder-of-key token. Unlike bearer tokens, the recipient of a holder-of-key token can verify whether the sender of the token is legitimate.
 
```

Comment:
Is it possible to add WARNING here that our client adapters currently doesn't do this verification, but we plan to add it in the future? 

I want to discuss with you first, but I believe it will be good if our client adapters have support for this? I think the specs mentions it on few places that ResourceServer should verify the access tokens and IMO this will be one of important advantages of this feature. WDYT? We can create separate JIRA for this and it can be done as followup once this PR is merged.